meet_balls_app/tests.py

```diff
@@ -28,7 +28,7 @@ def test_display_game_event_form(self, client):
 @pytest.mark.django_db
 class Testlogin:
     def test_successful_login(self, client, player):
-        username = "daniel"
+        username = "Playerr"
```

Comment:
same here should it be `Player`?

Reply:
No, I think someone did it randomly and I just changed it for was in Mose of the files, I can give it more meaningful name.

Reply:
We gave it the previous name "daniel" because we wanted to test successful_login with a user that is already at the system. @Danielsio created the Player fixture at the `conftest.py` file with the name "daniel".